### DISCUSSION TITLE
How to get galaxy brain badge.

### DISCUSSION THREAD

**Discussion by 4lph0nse** [Schweinepriester/github-profile-achievements#677] [Q&A] [answered]
How can I easily get 2 Accepted Answers to get the galaxy brain badge?

**Comment by andrepichardo** [comment, marked as answer]
To get the Galaxy Brain achievement badge, you need to fulfill the following criteria:

1. Discussions should be in a public repository
2. Get marked as answers to 2 or more questions
3. The questions cannot be asked by you. In short, you cannot answer your own questions, those do not count.
4. In fact, answering your own questions could prevent the achievement from displaying in your profile page.

Hope these steps help you in achieving your badge! Good luck 👍
